xrpl/models/transactions/signer_list_set.py

```diff
@@ -97,11 +97,19 @@ def _get_errors(self: SignerListSet) -> Dict[str, str]:
         if self.signer_entries is None:  # deletion of the SignerList object
             return errors
 
-        if self.signer_quorum <= 0:
+        if self.signer_quorum == REQUIRED:
```

Comment:
Does this work if the object is a dictionary? (I've never seen the REQUIRED keyword before, and my attempts to google it are not working well because it's such a common word - do you know how to find the docs for it / what it specifically does?)

Reply:
`REQUIRED` is an xrpl-py specific type, for all the required fields. There shouldn't be any issue regardless of what type the object is.